src/js/controller/controller.js

```diff
@@ -289,7 +289,7 @@ define([
                 if (_model.get('playOnViewable')) {
                     if (viewable) {
                         _autoStart();
-                    } else if (utils.isMobile()) {
+                    } else if (utils.isMobile() && model.get('mute')) {
```

Comment:
Maybe `_checkPlayOnViewable` should be handled by it own listener that is removed after the first unmute (see controls.js `unmuteAutoplay` and `unmuteCallback`).